### PR TITLE
added dante mccormick

### PR DESCRIPTION
**Pull request by KaiDevrim**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [✅] I have read the instructions on the README file before submitting my application. 
- [✅] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [✅] I have used the Markdown file template to add my information to the Year Book.
- [✅] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [✅] My application belongs to:
  - [✅] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [✅] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [✅] I understand I should not tag or add a reviewer to this Pull Request.
- [✅] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [✅] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by KaiDevrim** [comment]
I changed my github name from KaiSakurai to Kaidevrim

**Comment by KaiDevrim** [comment]
@juanpflores Can you please recheck but for the username KaiSakurai, I changed my name a few days ago

**Comment by KaiDevrim** [comment]
@elisemoe ok but do i need to submit the evidence again? I have it on my mac and its hard for me to access.

**Comment by KaiDevrim** [comment]
@elisemoe @juanpflores I redid the form.

File: _data/kaisakurai/kaisakurai.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Dante McCormick
+institution: University Preparatory Academy
+profile_pic: kaisakurai.png
+quote: Failure is just a part of life because life without an f is just a lie.
+github_user: kaisakurai
```
Comment (Mohammed-Shoaib) [inline _data/kaisakurai/kaisakurai.md:6]: Incorrect GitHub username, please change it to `KaiDevrim`.

**Pull request closed without merging**